Src/Program.cs

```diff
@@ -148,7 +148,15 @@ private void RunInstall(string extensionPath)
                 var installableExtension = _extensionManager.CreateInstallableExtension(extensionPath);
                 var identifier = installableExtension.Header.Identifier;
                 UninstallSilent(identifier);
-                _extensionManager.Install(installableExtension, perMachine: false);
+
+                var perMachine = false;
+                if(installableExtension.Header.AllUsers != perMachine)
```

Comment:
Ah, I see what the problem is...

If you type, `if(true){` Visual Studio reformats it to `if (true) { }`. If however you type, `if(true == false){<ENTER>`, Visual Studio leaves it as `if(true == false)`. I'm testing this with 2017.

Would you know an appropriate person to route this to? 😉